File: C#/networking/dto/FilterDTO.cs
using System;

namespace networking.dto
{
    [Serializable]
    public class FilterDTO
    {
        private string _landmark;
        private int _startHour;
        private int _endHour;

        public FilterDTO(string landmark, int startHour, int endHour)
        {
            _landmark = landmark;
            _startHour = startHour;
            _endHour = endHour;
        }

        public virtual string Landmark
        {
            get
            {
                return _landmark;
            }
        }

        public virtual int StratHour
        {
            get
            {
                return _startHour;
            }
        }

        public virtual int EndHour
        {
            get
            {
                return _endHour;
            }
        }
    }
}

File: Java/TravelAgencyNetworking/src/main/java/travelAgency/network/dto/FilterDTO.java
package travelAgency.network.dto;

import java.io.Serializable;

public class FilterDTO implements Serializable {
    String landmark;
    int startHour;
    int endHour;
    public FilterDTO(String landmark, int startHour, int endHour) {
        this.landmark = landmark;
        this.startHour = startHour;
        this.endHour = endHour;
    }

    public String getLandmark() {
        return landmark;
    }

    public void setLandmark(String landmark) {
        this.landmark = landmark;
    }

    public int getStartHour() {
        return startHour;
    }

    public void setStartHour(int startHour) {
        this.startHour = startHour;
    }

    public int getEndHour() {
        return endHour;
    }

    public void setEndHour(int endHour) {
        this.endHour = endHour;
    }

    @Override
    public String toString() {
        return "FilterDTO{" +
                "landmark='" + landmark + '\'' +
                ", startHour=" + startHour +
                ", endHour=" + endHour +
                '}';
    }
}
